dev/stop-all.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'path';
+import { execSync } from 'child_process';
+
+const dialectsDirectory = path.resolve(__dirname, 'dialects');
+const buffered = execSync(`find ${dialectsDirectory} | grep dialects | grep stop.sh`).toString();
+const stopScripts = buffered.split(/\n/g);
+
+stopScripts.forEach(scriptFullFileName => {  
+  if (!scriptFullFileName){
+    return;
+  }
+  
+  console.log(`
+
+-------------------------------------------------------
+${scriptFullFileName}  
+-------------------------------------------------------
+  `);
+
+  console.group();
+  try {
+    execSync(`${scriptFullFileName}`, {stdio: 'inherit'});
+  }
+  catch(e){
+    console.error(e);
+  }
+  console.groupEnd();
+});
+
+
+// NOTE: there is opportunity here for a future revision to replace the shell script with JavaSCript
```

Comment:
```suggestion
// NOTE: there is opportunity here for a future revision to replace the shell script with JavaScript

```

Not sure if the empty line would display correctly, but end with a newline